core/src/main/java/org/elasticsearch/bootstrap/Bootstrap.java

```diff
@@ -297,6 +298,13 @@ static void init(String[] args) throws Throwable {
         }
     }
 
+    @SuppressForbidden(reason = "Sets system properties passed as CLI parameters")
+    private static void elasticsearchSettings(Map<String, String> esSettings) {
+        for (Map.Entry<String, String> esSetting : esSettings.entrySet()) {
+            System.setProperty(esSetting.getKey(), esSetting.getValue());
```

Comment:
Why would we do this? I don't believe we do it currently, and I don't think we should be setting any sysprops from within java.